executor/admin_test.go

```diff
@@ -841,89 +917,91 @@ func (s *testSuite3) TestAdminCheckPartitionTableFailed(c *C) {
+		require.NoError(t, err)
 		err = tk.ExecToErr("admin check table admin_test_p")
-		c.Assert(err, NotNil)
-		c.Assert(err.Error(), Equals, fmt.Sprintf("[executor:8133]handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:<nil>", i+8, i+8))
+		require.EqualError(t, err, fmt.Sprintf("[executor:8133]handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:<nil>", i+8, i+8))
 		// TODO: fix admin recover for partition table.
-		txn, err = s.store.Begin()
-		c.Assert(err, IsNil)
+		txn, err = store.Begin()
+		require.NoError(t, err)
 		err = indexOpr.Delete(sc, txn, types.MakeDatums(i+8), kv.IntHandle(i+8))
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		err = txn.Commit(context.Background())
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		tk.MustExec("admin check table admin_test_p")
 	}
 
 	// Table count = index count, but the index value was wrong.
 	for i := 0; i <= 5; i++ {
 		partitionIdx := i % len(tblInfo.GetPartitionInfo().Definitions)
 		indexOpr := tables.NewIndex(tblInfo.GetPartitionInfo().Definitions[partitionIdx].ID, tblInfo, idxInfo)
-		txn, err := s.store.Begin()
-		c.Assert(err, IsNil)
-		_, err = indexOpr.Create(s.ctx, txn, types.MakeDatums(i+8), kv.IntHandle(i), nil)
-		c.Assert(err, IsNil)
+		txn, err := store.Begin()
+		require.NoError(t, err)
+		_, err = indexOpr.Create(ctx, txn, types.MakeDatums(i+8), kv.IntHandle(i), nil)
+		require.NoError(t, err)
 		err = txn.Commit(context.Background())
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		err = tk.ExecToErr("admin check table admin_test_p")
-		c.Assert(err, NotNil)
-		c.Assert(err.Error(), Equals, fmt.Sprintf("[executor:8134]col c2, handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)}, compare err:<nil>", i, i+8, i))
+		require.EqualError(t, err, fmt.Sprintf("[executor:8134]col c2, handle %d, index:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)} != record:types.Datum{k:0x1, decimal:0x0, length:0x0, i:%d, collation:\"\", b:[]uint8(nil), x:interface {}(nil)}, compare err:<nil>", i, i+8, i))
 		// TODO: fix admin recover for partition table.
-		txn, err = s.store.Begin()
-		c.Assert(err, IsNil)
+		txn, err = store.Begin()
+		require.NoError(t, err)
 		err = indexOpr.Delete(sc, txn, types.MakeDatums(i+8), kv.IntHandle(i))
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		err = txn.Commit(context.Background())
-		c.Assert(err, IsNil)
+		require.NoError(t, err)
 		tk.MustExec("admin check table admin_test_p")
 	}
 }
 
-func (s *testSuiteJoinSerial) TestAdminCheckTableFailed(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
+func TestAdminCheckTableFailed(t *testing.T) {
+	t.Parallel()
```

Comment:
```suggestion
```
I think this test should be run in serial because `set @@tidb_redact_log=0;` changes the golang's global variable `errors.RedactLogEnabled`.

Reply:
already fix, PTAL